jwst/flatfield/tests/test_find_dispaxis.py

```diff
@@ -0,0 +1,34 @@
+"""
+Test for flat_field.find_dispaxis
+"""
+import numpy as np
+
+from jwst.flatfield import flat_field
+
+def test_find_dispaxis():
+
+    # Create a dummy array of wavelengths.  The array is mostly zero,
+    # but the non-zero values are clearly increasing more rapidly
+    # in the horizontal direction than in the vertical direction.
+    wl = np.zeros((30, 200), dtype=np.float32)
+    wl[16, :] = np.arange(200, dtype=np.float32)
+    wl[17, :] = np.arange(200, dtype=np.float32) + 0.01
+    wl[18, :] = np.arange(200, dtype=np.float32) + 0.02
+    wl[19, :] = np.arange(200, dtype=np.float32) + 0.03
+
+    dispaxis = flat_field.find_dispaxis(wl)
+    assert dispaxis == 1
+
+    # This is the code that was in combine_fast_slow and that find_dispaxis
+    # replaced.
```

Comment:
I think this can be removed as, unless  I'm missing something, it isn't testing the code itself and is replaced.